src/deprecated.js

```diff
@@ -22,10 +22,11 @@ jQuery.fn.extend( {
 		return arguments.length === 1 ?
 			this.off( selector, "**" ) :
 			this.off( types, selector || "**", fn );
-	},
-	isArray: Array.isArray
+	}
```

Comment:
Looks good, though for cleanliness may want to remove this blank line which wasn't there before.
👍